docs/install/docker/stacks.md

```diff
@@ -0,0 +1,58 @@
+# Docker Project Stacks
+
+A Project Stack defines a set of platform configuration options that will get
+applied to each project when created
+
+For container based deployment models, this covers three things:
+
+ - `memory` - the value to apply (in MB) to limit container to consuming.
+ - `cpu` - a value between 1 and 100 that is the % of a CPU core the container should be allowed to consume.
+ - `container` - this is the fully qualified name of the container to use. The default container built when following the install instructions is named `flowforge/node-red:latest`
+
+## Creating Containers
+
+There is an example `Dockerfile` and `package.json` in the `node-red-container` 
+directory. This will start with `nodered/node-red:latest` as it's base and then 
```

Comment:
Keeping in mind these docs are in the core repo, which directory is this pointing at? If it's a file in a repo, can we link to it?

Reply:
Yep, we can link to https://github.com/flowforge/docker-compose/tree/main/node-red-container (the build.sh will get removed as it's leaked in from my local env)